content/c-sharp/concepts/deque/deque.md

```diff
@@ -0,0 +1,109 @@
+---
+Title: 'Deque'
+Description: 'A deque is a type of data structure that allows insert and delete elements at both ends.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Deques'
+  - 'Data Structures'
+  - 'Data Types'
```

Comment:
```suggestion
  - 'Deques'
  - 'Data Structures'
  - 'Data Types'
```
Include only those tags that are already present in the[ tags.md list](documentation/tags.md), and if you want the relevant tag then you can add one. Here `Deque` is not already available in the tags.md list, you can add it.